### PR TITLE
make errorbackoff customizeable for yawol-controller

### PR DESCRIPTION
**Pull request by breuerfelix**

### CODE REVIEW

File: charts/yawol-controller/values.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@ yawolCloudController:
 
 yawolController:
   gardenerMonitoringEnabled: false
+  errorBackoffBaseDelay: 5ms
```
Comment (nschad) [inline charts/yawol-controller/values.yaml:25]:
```suggestion
  errorBackoffInitialDelay: 5ms
```

File: cmd/yawol-controller/main.go [hunk 2]
```diff
@@ -55,6 +57,8 @@ func main() {
 	var probeAddr string
 
 	var concurrentWorkersPerReconciler int
+	var errorBackoffBaseDelay time.Duration
```
Comment (nschad) [inline cmd/yawol-controller/main.go:60]:
```suggestion
	var errorBackoffInitialDelay time.Duration
```

**Review by nschad — COMMENTED** [review]
just a tiny nit. I would rename that to `errorBackoffInitialDelay` or maybe even `errorInitialBackOffDelay`, whatever sounds more correct :)

**Comment by breuerfelix** [comment]
@nschad the official kubernetes variable is called base delay thats why i named it like this :)

**Review by dergeberl — DISMISSED** [review]

File: charts/yawol-controller/templates/yawol-controller.yaml [hunk 3]
```diff
@@ -64,9 +70,6 @@ spec:
         args:
```
Comment (dergeberl) [inline charts/yawol-controller/templates/yawol-controller.yaml:70]: I think we should add this to all 3 controllers not only the lb and lbm controller

**Review by dergeberl — CHANGES_REQUESTED** [review]

**Review by dergeberl — DISMISSED** [review]

**Review by dergeberl — DISMISSED** [review]

**Review by dergeberl — APPROVED** [review]

**Pull request merged**